src/pages/components/PageDetailsPage/PageDetailsPage.tsx

```diff
@@ -155,6 +155,7 @@ const PageDetailsPage: React.FC<PageDetailsPageProps> = ({
               {data.attributes.length > 0 && (
                 <Attributes
                   attributes={data.attributes}
+                  attributeValues={[]}
```

Comment:
Should this be optional with default value of `[]` instead

Reply:
same here